lib/new_relic/transaction/plug.ex

```diff
@@ -80,4 +82,30 @@ defmodule NewRelic.Transaction.Plug do
       _ -> nil
     end
   end
+
+  defp maybe_report_queueing(conn) do
+    [
+      queue_start_us: extract_queue_start_us(conn)
+    ]
+    |> NewRelic.add_attributes()
+  end
+
+  @queue_duration_headers ["x-request-start", "x-queue-start", "x-middleware-start"]
```

Comment:
I'd rather just stick with `x-request-start` than pile on a bunch of extra logic for backwards compatibility

Reply:
Happy to make that change. I didn't realize they were legacy